src/PAModel/SourceTransforms/AppTestTransform.cs

```diff
@@ -68,6 +75,9 @@ public void AfterRead(BlockNode control)
 
             foreach (var testStep in testStepsMetadata)
             {
+                // If ever encountering TestStepsMetadata, set empty to false
+                _entropy.TestStepsMetadataEmpty = false;
```

Comment:
This would be true.

Reply:
This would show that it's not empty, so wouldn't it be false...?

Reply:
oops sorry, so this statement should be inside first if condition and set the value to true